test/trento_web/controllers/v1/host_controller_test.exs

```diff
@@ -384,4 +384,29 @@ defmodule TrentoWeb.V1.HostControllerTest do
       )
     end
   end
+
+  describe "forbidden routes" do
```

Comment:
Just remove this. The exact same test is just above, under `forbidden routes` with some updates

Reply:
done